src/main/javascript/service/cat-validation-service.js

```diff
@@ -15,54 +17,122 @@ angular.module('cat.service.validation', ['cat.service.message'])
  * @description
  * value holding 'global' and 'field' errors
  */
-    .value('catValidations', {fieldErrors: {}})
-
-    /**
-     * @ngdoc service
-     * @name cat.service.validation:catValidationService
-     *
-     * @description
-     * Service which maps the 'fieldErrors' list recieved from the backend to a usable map for the client
-     */
-    .service('catValidationService', function CatErrorHttpInterceptor($globalMessages, catValidations) {
-        this.updateFromRejection = function (rejection) {
-            delete catValidations.global;
+    .value('catValidations', {
+        global: undefined,
+        fieldErrors: {}
+    })
 
-            if (!!rejection.data.globalErrors) {
-                catValidations.global = rejection.data.globalErrors;
-                $globalMessages.addMessages('error', rejection.data.globalErrors);
+/**
+ * @ngdoc object
+ * @name cat.service.validation:catValidationContexts
+ *
+ * @description
+ * value holding 'global' and 'field' errors
+ */
+    .value('catValidationContexts', {})
+
+/**
+ * @ngdoc service
+ * @name cat.service.validation:catValidationService
+ *
+ * @description
+ * Service which maps the 'fieldErrors' list recieved from the backend to a usable map for the client. All methods
+ * have a 'context' parameter as the last parameter. If no context is provided the global context will be used,
+ * otherwise the field error messages will be assigned to the specified context.
+ */
+    .service('catValidationService', function CatErrorHttpInterceptor($rootScope, $globalMessages, catValidations, catValidationContexts, catMessagesConfig) {
+
+        this.getValidations = function (context) {
+            if (context !== undefined) {
+                var validations = catValidationContexts[context];
+                if (validations === undefined) {
+                    throw new Error('Unknown context: ' + context);
+                }
+                return validations;
+            } else {
+                return catValidations;
             }
+        };
+
+        this.createContext = function () {
+            var uuid = window.cat.util.generateUUID();
+            catValidationContexts[uuid] = {
+                uuid: uuid,
+                global: undefined,
```

Comment:
this is dangerous -> see comment below [https://github.com/Catalysts/cat-angular/pull/37/files#diff-917ef52d37b2e2a8b6f242de4aeb8694R117]